rust/operator-binary/src/config.rs

```diff
@@ -98,6 +99,9 @@ pub enum Error {
 
     #[snafu(display("failed to build the OIDC wellkown path"))]
     InvalidOidcWellknownPath { source: url::ParseError },
+
+    #[snafu(display("Nifi doesn't support skipping the OIDC TLS verification"))]
+    NoOidcTlsVerificationNotSupported {},
```

Comment:
Thanks for adding the error! Can you please include the AuthClass in the error, similar to https://github.com/stackabletech/nifi-operator/blob/c61a0c0f63dc61854e72f62a363da747b0c3c5bd/rust/crd/src/authentication.rs#L34-L37?